laske_export/document/sales_order.py

```diff
@@ -47,6 +47,8 @@ def from_contact(self, contact):
         if not contact:
             return
 
+        self.customer_ovt = self.info_customer_ovt = contact.electronic_billing_address
```

Comment:
I'd prefer not to chain the assignment.

Reply:
Stylistic point taken. I suppose it's a bit less clear when chained.